### DISCUSSION TITLE
Grafana login

### DISCUSSION THREAD

**Discussion by BOOMEL1** [WLAN-Pi/feedback#71] [WLAN Pi OS]
I have just received the r4. I am trying to use Grafana which is asking for username and password. I have created a Grafana cloud ID and tried to use this but it will not accept it. Any ideas what I need to put in please? 

BOOMEL

**Comment by jiribrejcha** [comment]
wlanpi/wlanpi should do the trick. You can log in and adjust settings as needed. 

![image](https://github.com/user-attachments/assets/864a6237-08a0-40f6-b635-a5c22a2ad456)

**Comment by BOOMEL1** [comment]
That did the trick thanks
